Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM quay.io/condaforge/miniforge3
 
 # if you forked pandas, you can pass in your own GitHub username to use your fork
 # i.e. gh_username=myname
-ARG gh_username=pandas-dev
+ARG gh_username=myname
```

Comment:
Please change back

Reply:
will you please brief me about what to do now, as i am new to open source.
How to run precommit tests and what if got error in it while running doctests.